core/server/data/meta/index.js

```diff
@@ -66,7 +66,7 @@ function getMetaData(data, root) {
     if (settingsCache.get('logo')) {
         metaData.blog.logo.url = utils.url.urlFor('image', {image: settingsCache.get('logo')}, true);
     } else {
-        metaData.blog.logo.url = utils.url.urlJoin(utils.url.urlFor('admin'), 'img/ghosticon.jpg');
+        metaData.blog.logo.url = utils.url.urlJoin(utils.url.urlFor('admin'), 'assets/img/ghosticon.jpg');
```

Comment:
> Therefore, we'd either need to upload a default Ghost favicon in `/assets/img` for Admin

This would be my preference, there's no guarantee that the current `../favicon.ico` src used in Ghost-Admin will work because it's possible to set an entirely separate admin URL